### ISSUE TITLE
Opening nerdtree with :NERDTree shows `Press ENTER or type command to continue` after opening it

### ISSUE THREAD

**Issue by darthdeus** [preservim/nerdtree#1336]
#### Environment
- Operating System: Linux
- Vim/Neovim version `:version`: neovim 0.8.1
- NERDTree version, found on first line of quickhelp `?`: 6.10.16
- Are you using any of these NERDTree-dependent plugins? no

#### Steps to Reproduce the Issue
1. run `:NERDTree`

#### Current Behavior (Include screenshots where appropriate.)

A `Press ENTER or type command to continue` message appears, and NERDTree opens only after hitting ENTER

![image](https://user-images.githubusercontent.com/123374/206880538-ce2fae6d-1aa4-47f9-a42f-83047208cb74.png)

#### Expected Result

NERDTree opens immediately. This used to work until about a month or two ago, but seems the behavior changed. Commenting out the following line fixes the issue https://github.com/preservim/nerdtree/blob/master/autoload/nerdtree.vim#L221

Maybe I'm misunderstanding the purpose of this call, but shouldn't it rather be

```
function! nerdtree#echo(msg) abort
    redraw
    if !empty(a:msg)
        echomsg 'NERDTree: ' . a:msg
    endif
endfunction
```

to avoid echoing an empty message when there's nothing to show?

**Comment by imLew** [comment]
I can't say for sure but I think with the change you propose the behaviour of that function would change. Afaict right now it echoes as empty message if the argument is an empty message, and other prepends "NERDtree: " to the argument and then echoes it. With your change it would only do the latter and simply do nothing when the argument is empty.

Anyway, for me calling `:NERDtree` works as expected. I am also using neovim v0.8.1 and NERDtree 6.10.16. Did you try with all other plugins disable? e.g. with `neovim -u testrc` where `testrc` is a file that only loads NERDtree.

**Comment by unplugred** [comment]
Not sure if that's what the author is experiencing, but I've been getting similar problems when `set cmdheight=0` or `lua vim.o.ch=0` is set in vimrc

**Comment by rzvxa** [comment]
This problem isn't specific to NERDTree and is unreproducible there is most likely another issue in your configuration, I'm going to close this issue since it's been a while since the last comment here feel free to reopen it with a minimal Vim/Neovim configuration that can reproduce this issue.
